README.md

```diff
@@ -1,6 +1,23 @@
 # Welcome to Crowbar Angular Prototype
 [![Build Status](https://travis-ci.org/crowbar/crowbar-ui.svg?branch=master)](https://travis-ci.org/crowbar/crowbar-ui)
 
+### Setting up a development environment with Docker
+
+1. make sure `docker` is installed and the `docker` daemon is running.
+2. build the image from the [Dockerfile](Dockerfile):
+```
+docker build -t crowbar-ui .
+```
+3. run the container:
+```
+docker run -d --name "crowbar-ui_container" -v $(pwd)/assets:/crowbar-ui/assets -p 3000:3000 crowbar-ui
+```
+4. run the `gulp` watchdog:
+```
+docker exec -ti crowbar-ui_container gulp
+```
+5. work on the assets folder in your local git clone and the changes will be immediately visible in the browser.
```

Comment:
assets is only one of the folders required for this project. If the container will run the web server, then we need to also synchronise routes folder and app.js file to touch the mocked endpoints and their responses.
Also files in the root level provides the configuration for development and infrastructure related work. package.json and bower.json files are crucial to include new dependencies. How will them be synchronised?

Also, how are unit tests going to be run here, if the dependencies are installed in the container, not the host? And we need access to the coverage folder in order to get the coverage reports.